prow/config.yaml

```diff
@@ -643,10 +643,11 @@ presubmits:
     run_if_changed: 'gubernator'
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gubernator:0.2
-        env:
-        - name: GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
I added `--service-account=/etc/service-account/service-account.json`. e2e jobs references GOOGLE_APPLICATION_CREDENTIALS, gubernator should not need the specific env I think?